tests/test_cases.py

```diff
@@ -36,6 +38,19 @@ def run(*argv, **kwargs):
     return subprocess.run(argv, **kwargs)
 
 
+def init_git(repo_path):
+    run('git', 'init', cwd=repo_path)
+
+    run('git', 'config', 'merge.rpm-spec.name', 'RPM spec file merge driver', cwd=repo_path)
+    run('git', 'config', 'merge.rpm-spec.driver', f'{TOOL} %O %A %B %L %P', cwd=repo_path)
+    run('git', 'config', 'merge.rpm-spec.recursive', 'text', cwd=repo_path)
+    run('git', 'config', 'merge.verbosity', '5', cwd=repo_path)
+    run('git', 'config', 'merge.conflictStyle', 'diff3', cwd=repo_path)
+    run('git', 'config', 'user.name', 'Merge Driver User', cwd=repo_path)
```

Comment:
Curious question: 💁 was removed from here. Is that on purpose?

Reply:
Since #8, the name here doesn't matter. The changed name doesn't appear in actual tests.